job-dsls/jobs/kie/main/deploy_jobs.groovy

```diff
@@ -192,7 +192,7 @@ for (repoConfig in REPO_CONFIGS) {
             }
         }
 
-        if (repo == "optaplanner") {
+        if (repo == "process-migration-service") {
```

Comment:
Does this change remove the job or does it switch its Java version back to 8, which will fail for the `optaplanner` `main` branch?

Reply:
The optaplanner main branch deploy job is existing and on Jenkins [deployedRep](https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/main/job/deployedRep/). This will not be removed but it is not any more in the DSL groovy. So it is executing when you do a merge in main branch, and this has jdk11 (kie-jdk11). 
PIM was added a few weeks ago and was still built with `kie-jdk1.8`.
If you can see in this folder there is:

- **optaplanner** - main branch (kie-jdk11)
- **optaplanner-7.x** (kie-jdk1.8)
- **optaweb-vehicle-routing-7.x** (kie-jdk1.8)
- **optaweb-employee-rostering-7.x** (kie-jdk1.8)
and now **process-migration-service** (kie-jdk11)